### DISCUSSION TITLE
Github Actions condition doesn't work

### DISCUSSION THREAD

**Discussion by msashkin** [Mergifyio/mergify#1667] [Q&A]
Hi all,

I'm trying to setup auto-merge on success condition of my GitHub Actions Job.

Config file: https://github.com/odessajavaclub/coffee-pot/blob/master/.mergify.yml
CI GitHub Actions job: https://github.com/odessajavaclub/coffee-pot/blob/master/.github/workflows/workflow.yml
Example PR run: https://github.com/odessajavaclub/coffee-pot/pull/77/checks?check_run_id=1614443332

I followed https://docs.mergify.io/conditions.html#github-actions and defined my condition as:

```
      - check-success=Workflow CI
```

but it doesn't detect the defined job's status leading to the false condition and no merging at all.

Can you please help me to define the root cause of that and help fixing the mergify config file.

Thanks.

**Comment by jd** [comment]
Hi @msashkin!

The name of your check is `build` see https://docs.mergify.io/conditions.html#github-actions for reference.

**Reply by msashkin** [reply]
Thanks!

That solved my issue.
